tests/kani/FunctionStubbing/main.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// kani-flags: --harness main --enable-unstable --enable-stubbing
+//
+//! This tests stubbing for local functions and methods.
+
+fn local_fn() -> u32 {
+    0
+}
+
+fn the_answer() -> u32 {
+    42
+}
+
+mod local_mod {
+    pub fn pub_fn() -> u32 {
+        0
+    }
+
+    pub fn fn_delegating_to_priv_fn() -> u32 {
+        priv_fn()
+    }
+
+    fn priv_fn() -> u32 {
+        0
+    }
+
+    fn the_answer() -> u32 {
+        42
+    }
+}
+
+struct LocalType {}
+
+impl LocalType {
+    pub fn new() -> Self {
+        Self {}
+    }
+
+    pub fn pub_fn(&self) -> u32 {
+        0
+    }
+
+    pub fn fn_delegating_to_priv_fn(&self) -> u32 {
+        self.priv_fn()
+    }
+
+    fn priv_fn(&self) -> u32 {
+        0
+    }
+
+    fn the_answer(&self) -> u32 {
+        42
+    }
+}
+
+// TODO: Split up these assertions into separate harnesses, once stubbing is able to support that.
```

Comment:
Can you break this test into multiple ones? May local functions, private functions and methods?

Reply:
Done!